specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```
@@ -0,0 +1,2703 @@
+        "errors": {
+          "description": "Gets or sets the errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobErrorObject"
+          }
+        }
+      }
+    },
+    "TrainingJobTaskResultObject": {
+      "required": [
+        "trainedModelLabel",
+        "trainStatus"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModelLabel": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "trainStatus": {
+          "description": "Represents model train status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        },
+        "evaluationStatus": {
+          "description": "Represents model evaluation status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        }
+      }
+    },
+    "TrainingJobStatusResultStatusObject": {
+      "required": [
+        "percentComplete",
+        "elapsedTime"
+      ],
+      "type": "object",
+      "properties": {
+        "percentComplete": {
+          "format": "int32",
+          "description": "Represents progress percentage.",
+          "type": "integer"
+        },
+        "elapsedTime": {
+          "description": "Represents the elapsed time.\n            This string is in the format hh:mm:ss",
+          "type": "string"
+        }
+      }
+    },
+    "TrainingJobStatusObjectList": {
+      "required": [
+        "jobs",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "jobs": {
+          "description": "Gets or sets the jobs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrainingJobResultObject"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the next page link.",
+          "type": "string"
+        }
+      }
+    },
+    "TrainedModelObjectList": {
+      "required": [
+        "trainedModels",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModels": {
+          "description": "Gets or sets the jobs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrainedModelObject"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the next page link.",
+          "type": "string"
+        }
+      }
+    },
+    "TrainedModelObject": {
+      "required": [
+        "label",
+        "modelId",
+        "lastTrainedDateTime"
+      ],
+      "type": "object",
+      "properties": {
+        "label": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "modelId": {
+          "description": "Represents the Id of this model.\n            The model Id has the format \"{version}-{yyyyMMddThhmmss}-{Guid.NewGuid()}\".\n            This is used almost as a thumbprint for the model and isn't consumed in any of the APIs.",
```

Formatting won't carry through in generated code, so just normalize it:
```suggestion
          "description": "Represents the Id of this model. The model Id has the format \"{version}-{yyyyMMddThhmmss}-{Guid.NewGuid()}\". This is used almost as a thumbprint for the model and isn't consumed in any of the APIs.",
```

---

Same applies to any other intended formatting. The Azure SDK team is working on a new service definition language that should solve this, but it's not supported currently.